src/databricks/labs/remorph/snow/snowflake.py

```diff
@@ -197,8 +197,12 @@ def _parse_tonumber(args: list) -> local_expression.ToNumber:
 
     match len(args):
         case 1:
+            logger.warning("Precision and Scale are not specified, assuming defaults `precision`[38] and `scale`[0]")
+            logger.warning("No Format is specified, it will be inferred as simpled cast as decimal")
```

Comment:
```suggestion
            logger.warning("Precision and Scale are not specified, assuming defaults `precision`[38] and `scale`[0]. No Format is specified, it will be inferred as simpled cast as decimal")
```
don't log two log events one after another

Reply:
Done